Source/BugsnagConfiguration.h

```diff
@@ -84,6 +84,13 @@ typedef NSDictionary *_Nullable (^BugsnagBeforeNotifyHook)(
     NSArray *_Nonnull rawEventReports, NSDictionary *_Nonnull report);
 
 @interface BugsnagConfiguration : NSObject
+
+/**
+ * Determine the apiKey-validity of a passed-in string:
+ * Exactly 32 hexadecimal digits.
+ */
++ (BOOL)isValidApiKey:(NSString *_Nullable)apiKey;
```

Comment:
Would this method be publicly accessible? If so, I think we should attempt to hide this method as otherwise we'd have to keep the method signature public until the next major version.

Reply:
Good catch, thanks.  Moved to an `@interface` section in `BugsnagEvent.m` to limit the visibility.